### ISSUE TITLE
How to get the CoE File storage size down?

### ISSUE THREAD

**Issue by sabrineayadi** [microsoft/coe-starter-kit#8854]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Hi,
After the recent upgrade of CoE starter kit in the sandbox. environment, the file storage capacity has increased 
How can i reduce it, please?
Thank you in advance for your help! 
<img width="787" alt="Capturestorage" src="https://github.com/user-attachments/assets/01bc70fb-238b-4e41-9122-73134b44fc3b">

 

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

July 2024

### What app or flow are you having the issue with?

N/A

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
Hello. I do see this as well but only in environments where I had ALM issues with the kit. 
There have been lots of issues recently for kit users around upgrade which were product issues around upgrade dependency ordering. 
From looking at the link that is shown there it tends to be around "lots of customizations". We do not use any of the other features it references. 
Have you had such issues with upgrade of the kit recently?

**Comment by sabrineayadi** [comment]
Hello !
I encountered the following issues on August 4th after upgrading to the July version.
![image](https://github.com/user-attachments/assets/19ab4834-deba-416b-bf58-14ba535b64c0)

I then upgraded to the August version, but since August 7th.
<img width="174" alt="Captureadmin" src="https://github.com/user-attachments/assets/bd1b4d3d-04fd-4557-ad48-8227f2317341">
 I've noticed a significant increase, particularly with async operations.
<img width="793" alt="Capturegrpahique" src="https://github.com/user-attachments/assets/1e9dcd61-a983-4cb7-b1b0-57b0d9c2a62a">

**Comment by sabrineayadi** [comment]
<img width="934" alt="allsolutions" src="https://github.com/user-attachments/assets/4328b8a7-f4f1-4057-8b96-4ba656ae8a94">

**Comment by Jenefer-Monroe** [comment]
Oh I see, you installed the preview build into your main environment. Those are released only for early assistance in testing. 

Can you please upgrade to the August release? https://github.com/microsoft/coe-starter-kit/releases/tag/CoEStarterKit-August2024

I don't think that will resolve the issue for you but you will be on a version of the kit that is intended for use. 

Before you do so though, please go look at your solution history and look for _CoreComponents_. Do you have recent failures in upgrade?
![image](https://github.com/user-attachments/assets/92617dab-b1d4-4106-a23c-da38db4b75c0)
![image](https://github.com/user-attachments/assets/a5e2cd27-ef6b-4d1a-a24b-cb0ec78b60d2)

**Comment by sabrineayadi** [comment]
Everything is fine in the history
<img width="824" alt="hisotysolution" src="https://github.com/user-attachments/assets/cd1e51d1-df34-45f8-8df4-88029bfd9b82">
 You sent the link in this release for this task https://github.com/microsoft/coe-starter-kit/issues/8611.
 My goal is to reduce storage, as the client sent me an alert and I need to quickly find a solution.
 Thank you for your help!

**Comment by Jenefer-Monroe** [comment]
I see, but I also asked you to not install that build in your production environment. 
![image](https://github.com/user-attachments/assets/154ad405-51c6-469b-af1f-3d15d6baf791)

**Comment by Jenefer-Monroe** [comment]
Unfortunately finding a resolution here is going to take a while. This is a product bug as we do not use any of the features that product table stores. So I will need to work internally to find assistance from development teams

It would be helpful if you can create a support ticket on behalf of your client.

**Comment by sabrineayadi** [comment]
We're facing an issue in the sandbox environment while testing the feature. If we upgrade to the version mentioned in your last comment, it might help reduce capacity

**Comment by Jenefer-Monroe** [comment]
I dont think so no. I have some tests running. I dont know why some environments are hitting this and not others. 
How old is the environment?

**Comment by sabrineayadi** [comment]
11 months. Could you kindly share your MSFT contact with me?

**Comment by Jenefer-Monroe** [comment]
If support pushes back you can have them reach out to Grant Archibald who is the PM for the kit. 
He is on vacation but will be back tomorrow

**Comment by Jenefer-Monroe** [comment]
Hiding some comments to make this easier to track

**Comment by Jenefer-Monroe** [comment]
This table evidently stores customizations, which aside from roll up fields is the only thing I can think of that would interact with the table from the kit.

The upticks seem to correspond with any upgrade operation. Regardless of their size. In this latest test case I over wrote the solution entirely and still it caused an uptick. 

![image](https://github.com/user-attachments/assets/2ff843dd-844d-45e6-83c7-46a54cad300a)

**Comment by sabrineayadi** [comment]
In this case, do you have any recommendations for reducing capacity?

**Comment by Jenefer-Monroe** [comment]
Sorry by I don’t understand the issue yet. This is a product issue and we have begun engagement with product team, however if you repro the issue please create a support ticket that will help us drive the matter

**Comment by pnoot7** [comment]
Is the CoE team suggesting we open tickets directly with MS Support on this? Typically, they refuse to touch anything related to CoE.

The recommended steps in the article, https://learn.microsoft.com/en-us/power-platform/admin/cleanup-asyncoperationbase-table?WT.mc_id=ppac_inproduct_env, do not delete any records, even if you leave off the option parameters.

**Comment by Ayesha-320** [comment]
Hi Team,
According to https://learn.microsoft.com/en-us/power-platform/alm/pipelines, Power Platform Pipelines are now the recommended approach for Application Lifecycle Management (ALM). The ALM Accelerator is no longer actively maintained or supported. Therefore, this issue is being closed accordingly.  

Best Regards, 
CoE Starter Kit Team

**Comment by pnoot7** [comment]
What does that have to do with this issue? Please reopen. This is for the CoE install, which certainly does not use pipelines.

**Comment by pnoot7** [comment]
https://github.com/microsoft/coe-starter-kit/issues/9577
